credential/store/src/main/java/org/wildfly/security/credential/store/impl/VaultObjectInputStream.java

```diff
@@ -31,9 +31,9 @@ final class VaultObjectInputStream extends ObjectInputStream {
 
     protected Class<?> resolveClass(final ObjectStreamClass desc) throws IOException, ClassNotFoundException {
         final String name = desc.getName();
-        switch (name) {
-            case SecurityVaultData.PICKETBOX_CLASS_NAME: return SecurityVaultData.class;
-            default: return super.resolveClass(desc);
+        if (name.equals(SecurityVaultData.PICKETBOX_CLASS_NAME)) {
+            return SecurityVaultData.class;
         }
+        return super.resolveClass(desc);
```

Comment:
we can actually simplify this further by using a ternary statement:
```suggestion
        if (name.equals(SecurityVaultData.PICKETBOX_CLASS_NAME)) {
            return SecurityVaultData.class;
        }
        return name.equals(SecurityVaultData.PICKETBOX_CLASS_NAME) ?  SecurityVaultData.class : super.resolveClass(desc);
```

Reply:
*edited above to exclude the if statement